src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -3,9 +3,10 @@ import {Text, View} from 'react-native';
 import Tooltip from 'react-native-walkthrough-tooltip';
 import {TouchableWithoutFeedback} from 'react-native-gesture-handler';
 import MemoIcQuestionMark from '../../../assets/icons/Ic_question_mark';
-import {normalize} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
 import {CircleGradient} from '../../../components/Karma/CircleGradient';
 import {BetterSocialLogoGram} from '../../../assets';
+import { COLORS } from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement on line 6 has been updated to include `fonts` from the `../../../utils/fonts` path. This is a good change as it allows for more consistent typography across the application by using predefined font styles. However, please ensure that all instances where fonts are used have been updated to use this new import.

```diff
- import {normalize} from '../../../utils/fonts';
+ import {fonts, normalize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->